egs/tedlium3/ASR/local/compile_hlg.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
```

Comment:
Could you replace it with a symlink to the file in librispeech/ASR/local/compile_hlg.py?
And also for the following files that are seldomly changed?
- egs/tedlium3/ASR/local/compute_fbank_musan.py
- egs/tedlium3/ASR/local/convert_transcript_words_to_bpe_ids.py
- egs/tedlium3/ASR/local/convert_transcript_words_to_bpe_ids.py
- egs/tedlium3/ASR/local/convert_transcript_words_to_tokens.py
- egs/tedlium3/ASR/local/generate_unique_lexicon.py
- egs/tedlium3/ASR/local/prepare_lang.py
- egs/tedlium3/ASR/local/prepare_lang_bpe.py
- egs/tedlium3/ASR/local/test_prepare_lang.py
- egs/tedlium3/ASR/local/train_bpe_model.py
- egs/tedlium3/ASR/transducer_stateless/beam_search.py
- egs/tedlium3/ASR/transducer_stateless/conformer.py
- egs/tedlium3/ASR/transducer_stateless/decoder.py
- egs/tedlium3/ASR/transducer_stateless/encoder_interface.py
- egs/tedlium3/ASR/transducer_stateless/joiner.py
- egs/tedlium3/ASR/transducer_stateless/model.py
- egs/tedlium3/ASR/transducer_stateless/subsampling.py
- egs/tedlium3/ASR/transducer_stateless/test_decoder.py
- egs/tedlium3/ASR/transducer_stateless/transformer.py

If we ever need to modify it for a specific scenario, we can just copy it and then modify the copy.

Reply:
OK, for the unchanged files, I will use symlinks.